### PR TITLE
Added a list of power users

### PR DESCRIPTION
**Pull request by joemoe**
## Summary of the changes / Why this is an improvement

- To ensure the compatibility of CrateDB with external tools, we are introducing *Power Users*.
- They have running setups for those tools, can answer questions, and check compatibility on updates.

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -14,6 +14,37 @@ contributions!
...
+* Keep track of releases, updates and developments of the tools.
+* Test integration after minor and mayor CrateDB releases.
+* Test integration after tool releases.
+* Have a working setup available including a quick demo.
+* Being able to answer questions for the specific tool.
+* Ensuring guides are up to date.
+
+
+Power Users (*Tool, User, Tool Version // CrateDB Version, Issues*) :
+------------
```
Comment (mechanomi) [inline README.rst:33]:
```suggestion
Power Users (*Tool, User, Tool Version // CrateDB Version, Issues*) :
---------------------------------------------------------------------
```

Comment (mechanomi) [reply README.rst:33]: however, I would drop this from the title and instead reformat the list as a table

Comment (joemoe) [reply README.rst:33]: tried to avoid that. :)

will do.

**Comment by joemoe** [comment]
> please reformat as a table

please have another look at this beauty @mechanomi

File: README.rst [hunk 2]
```diff
@@ -29,21 +29,54 @@ Responsiblities of Power Users:
...
+.. _Known Issues Grafana: https://github.com/crate/crate/labels/tool%3A%20Grafana
+.. _Known Issues PowerBI: https://github.com/crate/crate/labels/tool%3A%20PowerBI
+.. _Known Issues Tableau: https://github.com/crate/crate/labels/tool%3A%20Tableau
+.. _Known Issues Looker: https://github.com/crate/crate/labels/tool%3A%20Looker
+.. _Known Issues Node Red: https://github.com/crate/crate/labels/tool%3A%20Node-Red
+.. _Known Issues NiFi: https://github.com/crate/crate/labels/tool%3A%20NiFi
+.. _Known Issues R: https://github.com/crate/crate/labels/tool%3A%20R
+.. _Known Issues Azure Functions: https://github.com/crate/crate/labels/tool%3A%20Azure%20Functions
+.. _Known Issues Prometheus: https://github.com/crate/crate/labels/tool%3A%20Prometheus
+.. _Known Issues Adapter: https://github.com/crate/crate_adapter/issues
```
[the remark this answers is not in the document]
Comment (mechanomi) [reply README.rst]: this is addressed with my fixup commit

**Comment by amotl** [comment]
Hi.

Apparently, this list has already been integrated into the tools section [1]. We should compare it and afterwards close this PR.

With kind regards,
Andreas.

[1] https://crate.io/docs/crate/clients-tools/en/latest/#tools

**Comment by amotl** [comment]
Hi again,

@seut: Do you agree to close this PR?

With kind regards,
Andreas.

**Comment by hammerhead** [comment]
Just came across this old PR by chance and am taking the liberty to close it, as most of the mentioned users aren't with the company anymore and it has been open for over four years.

**Pull request closed without merging**